packages/view-types/feature-list/src/FeatureListOptions.js

```diff
@@ -34,17 +34,18 @@ export default function FeatureListOptions(props) {
   return (
     <OptionsContainer>
       {children}
-      <TableRow>
-        <TableCell className={classes.labelCell} htmlFor="feature-list-sort-option-select">
+      <TableRow key="feature-list-sort-option">
```

Comment:
What is the purpose of the `key`s? I thought keys were mainly intended for rendering arrays of components https://react.dev/learn/rendering-lists#rules-of-keys

Reply:
Yeah, I did a bit more reading and you are both right. I thought that they improve accessibility, looks like I got confused. I will remove them shortly, thanks for the catch.

Reply:
Done in commit 34a756fc